inc/link-roundups/class-save-to-site-button.php

```diff
@@ -55,14 +55,29 @@ public static function init() {
 	/**
 	 * Returns the link for the bookmarklet button.
 	 *
+	 * If get_shortcut_link() returns an empty string (WP 4.9+) and
+	 * if the https://wordpress.org/plugins/press-this/ plugin is installed,
+	 * this plugin will use the plugin's bookmarklet code instead.
+	 *
 	 * @since 0.3
 	 *
+	 * @uses get_shortcut_link
+	 * @uses https://github.com/WordPress/press-this/blob/690c99c8cd8feba4fe4fbfff89a955a325e35505/press-this-plugin.php#L118-L173
 	 * @return String. Javascript bookmarklet code.
 	 */
 	public static function shortcut_link() {
 
 		// This is the default 'Press This!' button link.
-		$shortcut_link = htmlspecialchars( get_shortcut_link() );
+		$shortcut_link = '';
+
+		if ( function_exists( 'get_shortcut_link' ) ) {
+			$shortcut_link = htmlspecialchars( get_shortcut_link() );
+		}
+
+		// since 4.9, get_shortcut_link has returned the empty string.
+		if ( empty( $shortcut_link ) && function_exists( 'press_this_get_shortcut_link' ) ) {
```

Comment:
@benlk Do you think it'd be beneficial here to also make sure the WP version is greater than 4.9?

Reply:
I don't think it's necessary; we're less concerned about what version of WordPress is in use and more concerned about whether that version of WordPress provides the link. Checking the WordPress version is a way of getting a clue as to whether the link would be provided, but it's easier to just test whether the link is provided.

Reply:
Update: https://github.com/INN/link-roundups/issues/166